test/colab/colab-test.mk

```diff
@@ -0,0 +1,31 @@
+.PHONY: colab-test-internal
```

Comment:
Can we move the `DOCKER_RUN` definition to this file? It is easier to read without jumping files and it is only used here.